rs/config/src/subnet_config.rs

```diff
@@ -160,6 +160,11 @@ pub const DEFAULT_UPLOAD_CHUNK_INSTRUCTIONS: NumInstructions = NumInstructions::
 pub const DEFAULT_CANISTERS_SNAPSHOT_BASELINE_INSTRUCTIONS: NumInstructions =
     NumInstructions::new(2_000_000_000);
 
+/// The cycle cost overhead of executing canister instructions when running in Wasm64 mode.
+/// This overhead is a multiplier over the cost of executing the same instructions
+/// in Wasm32 mode.
```

Comment:
Should we also mention a bit why there's an overhead? Basically the bounds check that adds it.

Reply:
makes sense, thanks. fixed now.